### PR TITLE
feat: add `cts_login`  and `cts_logout` system events

### PR DESCRIPTION
**Pull request by rrrrs09**
# Description

Added `cts_login`  and `cts_logout` system events which are sent by botx when user joins or leaves corporate server.

# Checklist

- [x] Linters and tests have passed
- [x] I've written a changelog for PR change
- [x] I'll make a release when PR is approved

### CODE REVIEW

File: botx/middlewares/ns.py [hunk 1]
```diff
@@ -176,6 +176,8 @@ def get_chain_key_by_message(message: Message) -> Tuple[str, UUID, UUID, UUID]:
     # key is a tuple of (host, bot_id, chat_id, user_huid)
     if message.user_huid is None:
         raise RuntimeError("key for chain can be obtained only for messages from users")
+
+    assert message.group_chat_id is not None
```
Comment (lxmnk) [inline botx/middlewares/ns.py:180]: I suggest raising `RuntimeError`, like above.

File: botx/models/events.py [hunk 2]
```diff
@@ -60,6 +60,25 @@ class InternalBotNotificationEvent(BotXBaseModel):
...
+
+
+class CTSLogoutEvent(BotXBaseModel):
+    """Shape for `system:cts_logout` event data."""
+
+    #: huid of user which logged out from CTS.
+    user_huid: UUID
+
+    #: CTS id.
+
```
Comment (lxmnk) [inline botx/models/events.py:80]: ???

File: botx/testing/building/entites.py [hunk 3]
```diff
@@ -144,7 +144,7 @@ def forward(
         """
         if message and not forward:
             forward = Forward(
-                group_chat_id=message.group_chat_id,
+                group_chat_id=message.group_chat_id,  # type: ignore
```
Comment (lxmnk) [inline botx/testing/building/entites.py:147]: Here you can add `assert`.

File: botx/middlewares/ns.py (hunk 1 above)
Comment (rrrrs09) [reply botx/middlewares/ns.py]: Done

File: botx/models/events.py (hunk 2 above)
Comment (rrrrs09) [reply botx/models/events.py:81]: Fixed

File: botx/testing/building/entites.py (hunk 3 above)
Comment (rrrrs09) [reply botx/testing/building/entites.py]: Done

**Review by lxmnk — APPROVED** [review]

**Review by Opti213 — APPROVED** [review]

**Pull request merged**